GG/GG/ListBox.h

```diff
@@ -323,12 +320,17 @@ class GG_API ListBox : public Control
         auto-scrolling. */
     unsigned int    AutoScrollInterval() const;
 
+    /** Return true if drops are allowed.*/
+    bool AllowingDrops();
+
+
     mutable ClearedSignalType        ClearedSignal;         /// the cleared signal object for this ListBox
     mutable BeforeInsertSignalType   BeforeInsertSignal;    ///< the before insert signal object for this ListBox
     mutable AfterInsertSignalType    AfterInsertSignal;     ///< the after insert signal object for this ListBox
     mutable SelChangedSignalType     SelChangedSignal;      ///< the selection change signal object for this ListBox
     mutable DroppedSignalType        DroppedSignal;         ///< the dropped signal object for this ListBox
     mutable DropAcceptableSignalType DropAcceptableSignal;  ///< the drop-acceptability signal object for this ListBox
+    mutable RowMovedSignalType       MovedSignal;           ///< the moved signal object for this ListBox
```

Comment:
The concern is for cases where the signature isn't apparent, such as when connecting the signal to a function by name.

The other signal names are mostly not ambiguous, though...  Perhaps `DroppedSignal` could refer to the `ListBox` being dropped, rather than being dropped on, but that wouldn't be my default assumption. With `MovedSignal`, it could easily mean the `ListBox` itself is being moved around.